### ISSUE TITLE
[testing] base accessibility testing for components

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon-addons-iot-react#1436]
### Summary

Any component that is not a stock re-export/proxied Carbon component should have a base accessibility `.toBeAccessible()` test for renders w/ default props. We expect that Carbon components we re-export without modifications will be tested for accessibility by the Carbon team.

Any violations should be fixed.

We can do this component by component so that each PR diff/review is isolated, small, and digestible.
